feature_tests/steps/ods_steps.py

```diff
@@ -0,0 +1,20 @@
+from behave import given
+from domain.core.error import DuplicateError
+from domain.core.root import Root
+
+
+def _add_ods_to_context(context, id: str, name: str):
+    if id in context.ods_organisations:
+        raise DuplicateError("Invalid test: ODS Organisation exists")
```

Comment:
mmm i don't this logic belongs here. What if we were genuinely testing the effect of creating the same org twice? I think that still falls under the realm of domain logic